### PR TITLE
[bug-fix] eventhub status value is now interpreted as http code

### PR DESCRIPTION
**Pull request by davidkhala**
instead of string like "NOT_FOUND", we see the actual value is 404
Signed-off-by: DavidLiu <david.yx.liu@oracle.com>

### CODE REVIEW

**Comment by davidkhala** [comment]
along the way, I want to migrate from a deprecated jsrsasign method. 

Reference is [here](https://github.com/kjur/jsrsasign/blob/f3d70198b2d923bdeacd2bd8c60605a96101d802/src/asn1csr-1.0.js#L325)

**Comment by bestbeforetoday** [comment]
> along the way, I want to migrate from a deprecated jsrsasign method.
> 
> Reference is [here](https://github.com/kjur/jsrsasign/blob/f3d70198b2d923bdeacd2bd8c60605a96101d802/src/asn1csr-1.0.js#L325)
> 
> **EDIT**: we find this method is new and does not exist in jsrsasign 8, rollback first

It would be great to move to the most current version of jsrsasign but API changes mean we're stuck at v8 unless the SDK code is updated to reflect the newer API

**Review by bestbeforetoday — CHANGES_REQUESTED** [review]
It's a shame no scenario tests pick up this problem as then we're sure we match runtime behaviour rather than just values injected by unit tests.

It would look clearer to me to use the protobuf definitions rather than hard-code the status values, so I think that's worth changing (see comments). Otherwise looks OK to me.

File: fabric-common/test/EventService.js [hunk 1]
```diff
@@ -766,7 +766,7 @@ describe('EventService', () => {
 			eventer1.waitForReady = sandbox.stub().resolves();
 			await eventer1.connect(endpoint);
 			eventService.blockType = 'full';
-			onStub.yields({Type: 'status', status: 'NOT_FOUND'});
+			onStub.yields({Type: 'status', status: 404});
```
[the remark this answers is not in the document]
Comment (davidkhala) [reply fabric-common/test/EventService.js:769]: Indeed, will do

**Comment by davidkhala** [comment]
> > along the way, I want to migrate from a deprecated jsrsasign method.
> > Reference is [here](https://github.com/kjur/jsrsasign/blob/f3d70198b2d923bdeacd2bd8c60605a96101d802/src/asn1csr-1.0.js#L325)
> > **EDIT**: we find this method is new and does not exist in jsrsasign 8, rollback first
> 
> It would be great to move to the most current version of jsrsasign but API changes mean we're stuck at v8 unless the SDK code is updated to reflect the newer jsrsasign API

Noted and will do it later.

**Review by bestbeforetoday — APPROVED** [review]

**Pull request merged**